files/validator.rb

```diff
@@ -0,0 +1,42 @@
+def handler
+  req = Nginx::Request.new
+  uri = req.uri # of form .*/small_light[^\/]*/.*
+  v = Nginx::Var.new
+  threshold = v.small_light_maximum_size
+
+  params, uri_redir = parse_uri(uri)
+  if threshold == "" # TODO not sure it's working
```

Comment:
Please describe why you think __"not sure it's working"__ at Pull Request comment.